code/modules/mob/mob_movement.dm

```diff
@@ -157,6 +157,8 @@
 						M.stop_pulling()
 
 		switch(mob.m_intent)
+			if("sprint")
+				move_delay += config.run_speed

```

Comment:
I think your sprinting speed should be affected by the damage on your legs or something. It would seem silly if you are on the edge of dying, with all the damage slowdowns and maybe a coldness slowdown, and you are able to sprint like it is nothing.